Projects/UOContent/Items/Weapons/Abilities/Disarm.cs

```diff
@@ -52,7 +47,7 @@ public override void OnHit(Mobile attacker, Mobile defender, int damage)
 
             var toDisarm = defender.FindItemOnLayer(Layer.OneHanded);
 
-            if (toDisarm?.Movable == false)
+            if (toDisarm is null || toDisarm?.Movable == false)
```

Comment:
I have never seen a `is null` check. What is the use case?
If you are trying to say, this is true if toDisarm == null || !toDisarm.Movable.
Then try this:
```cs
if (toDisarm?.Movable != true)
```

Reply:
This is needed because when toDisarm is null it doesnt checked of twohanded.